libraries/ble/src/iot_ble_gap.c

```diff
@@ -652,6 +652,15 @@ BTStatus_t IotBle_On( void )
             if( status == eBTStatusSuccess )
             {
                 status = prvSetAdvData( &_scanRespParams );
+
```

Comment:
The log may seem misleading here as `eBTStatusNoMem` can be emitted due to other reasons as well. Could you update the log accordingly.